htdocs/main.php

```diff
@@ -276,12 +276,12 @@ function tplFromRequest($param)
 }
 $tpl_data['userPerms']   = $realPerms;
 $tpl_data['studyParams'] = array(
-                            'useEDC'      => $config->getSetting('useEDC') ?
-        $config->getSetting('useEDC') : false,
-                            'useProband'  => $config->getSetting('useProband') ?
-        $config->getSetting('useProband') : false,
-                            'useFamilyID' => $config->getSetting('useFamilyID') ?
-        $config->getSetting('useFamilyID') : false,
+                            'useEDC'      => $config->getSetting('useEDC'),
+                            'useProband'  => $config->getSetting('useProband'),
+                            'useFamilyID' => $config->getSetting('useFamilyID'),
+                            'useConsent'  => $config->getSetting(
+                                'ConsentModule'
+                            )['useConsent'],
```

Comment:
@jstirling91 `useConsent` is not part of config table, hence it has to be fetched from `config.xml` for now and that's the way it is structured there:

```xml
<ConsentModule>
    <useConsent>true</useConsent>
    <Consent>
        <name>study_consent</name>
        <label>Consent to Study</label>
    </Consent>
</ConsentModule>
```

Which is why I had to get the parent first